app/main/views/suppliers.py

```diff
@@ -541,6 +564,33 @@ def duns_number():
     ), 200 if not errors else 400
 
 
+@main.route('/create/lookup', methods=['GET', 'POST'])
+def lookup():
```

Comment:
From a dev's angle at 1000ft, I _would_ have expected the `lookup` view to be the thing that does the _lookup_, rather than being the view that just does the confirmation.

Reply:
Could change the view function name to be `lookup_result`?

Reply:
`lookup_confirm` 👍